public/css/style.css

```diff
@@ -1,5 +1,5 @@
 body {
-    background-color: #B1D6D8;   
+    background-color: #b1d6d8;
```

Comment:
it's actually fine to use all upper-case or all lower-case as long as you're consistent. My VSCode just happens to be set to use all lower-case :)